RUST_CODING_STYLE.md

```diff
@@ -12,7 +12,7 @@ The `--check` flag can be used to check if the code is formatted correctly.
 ./scripts/rust_lint.sh
 ```
 
-> **Note:** xclippy is an alias for `cargo clippy` with additional flags to enable more lints.
+> **Note:** clippy is an alias for `cargo clippy` with additional flags to enable more lints.
```

Comment:
It's actually `xclippy` https://github.com/aptos-labs/aptos-core/blob/main/.cargo/config.toml#L2

Reply:
https://github.com/aptos-labs/aptos-core/commit/8dc501b12028082272c0ee29ebfaf2bbf3f830fc
Good call, fixed